File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/Cities/Pref.java
package com.appsbygu.kadai_weather_java.models.apis.Cities;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import io.realm.RealmList;
import io.realm.RealmObject;

@Root(strict = false)
public class Pref extends RealmObject {

    @Attribute(name="title", required = false)
    private String title;

    @ElementList(entry="city", inline = true, required = false)
    private RealmList<City> cities;

    public String getTitle ()
    {
        return title;
    }
    public RealmList<City> getCities ()
    {
        return cities;
    }

}

File: app/src/main/java/com/appsbygu/kadai_weather_java/models/apis/cities/Pref.kt
package com.appsbygu.kadai_weather_java.models.apis.cities

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

import io.realm.RealmList
import io.realm.RealmObject

@Root(strict = false)
open class Pref : RealmObject() {

    @field:Attribute(name = "title", required = false)
    var title: String? = null

    @field:ElementList(entry = "city", inline = true, required = false)
    var cities: RealmList<City>? = null

}